packages/asset-swapper/src/index.ts

```diff
@@ -70,5 +70,5 @@ export {
 } from './utils/market_operation_utils/types';
 export { affiliateFeeUtils } from './utils/affiliate_fee_utils';
 export { ProtocolFeeUtils } from './utils/protocol_fee_utils';
-export { QuoteRequestor, RfqtIndicativeQuoteResponse } from './utils/quote_requestor';
+export { QuoteRequestor, IndicativeQuote } from './utils/quote_requestor';
```

Comment:
Why not just export IndicativeQuote from `quote-server` directly?

Reply:
My logic was that we should import the type from the module that requires us to reference that type.  But if there's some convention here then I'm happy to follow it.

Reply:
Lines 1 through 16 of this file seem to follow the pattern of exporting directly from the package, so I'd prefer to do this here as well, i.e. `export { IndicativeQuote } from '@0x/quote-server`